### ISSUE TITLE
TypeLoadException: Could not resolve type with token 01000043 (from typeref, class/assembly UnityEngine.Touch, UnityEngine.CoreModule, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null) 

### ISSUE THREAD

**Issue by omersupersonic** [facebook/facebook-sdk-for-unity#639]
## Checklist

- [x] I've updated to the latest released version of the SDK
- [x] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-sdk-for-unity/issues)
- [x] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [x] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [x] This issue is not security related and can safely be disclosed publicly on GitHub

## Environment

Describe your dev environment here, giving as many details as possible. If you have them, make sure to include:

- Unity Editor Version: `X.X.X`
- Unity SDK Version: `X.X.X`
- Installation Platform & Verison: `[iOS|Android|WebGL]` version `X.X.X`

## Goals

What do you want to achieve?

## Expected Results

What do you expect to happen?

## Actual Results

What actually happened? Can you provide a stack trace?

## Steps to Reproduce

What are the steps necessary to reproduce this issue?

## Code Samples & Details

Please provide a code sample, as well as any additional details, to help us track down the issue. If you can provide a link to a test project that allows us to reproduce the issue, this helps us immensely in both the speed and quality of the fix.

**Note:** Remember to format your code for readability:

```java
// INSERT YOUR CODE HERE
```

**Comment by yourbestfriendton** [comment]
I ran into this with the pre-packaged FB Unity SDK 12.0.0 download on multiple unity versions and platforms. Even when creating an empty new Unity project and only importing the SDK and running the sample scene.

Unity Editor Version: 2020.3.24f1
Editor Platforms attempted: Mac, iOS

Unity Editor Version: 2020.3.12f1
Editor Platforms attempted: Mac, Android

Unity Editor Version: 2021.3.8f1
Editor Platforms attempted: Mac, Android 

Unity Editor Version: 2020.3.37f1
Editor Platforms attempted: Mac, Android

I was able to "fix" it by building the package locally, following the README.mdown instructions. I only verified that it solved the issue on 2020.3.24f1, since that is the actual Unity version I wanted to use.
